packages/utils/src/components/InjectClassName/InjectClassName.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Utils/Components/InjectClassName" />
+
+# InjectClassName
+
+`InjectClassName` can be used to add a className to all its children.
+
+#### Parameters
+
+It takes only one parameter `className` which is of type string.
+
+#### Example
+
+```tsx
+import { InjectClassName } from '@medly-components/utils';
```

Comment:
The major benefit of using `InjectClassName` is to create styles without providing the html tag to styled-components. Checkout this code -> https://github.com/medly/medly-components/blob/master/packages/icons/src/SvgIcon/SvgIcon.styled.tsx#L22